packages/terra-list/src/MultiSelectList.jsx

```diff
@@ -138,7 +138,7 @@ class MultiSelectList extends React.Component {
 
     return (event) => {
       if (event.nativeEvent.keyCode === KEYCODES.ENTER) {
-        if (item.props.isSelectable !== false && this.shouldHandleSelection(index)) {
+        if (item.props.isSelectable && this.shouldHandleSelection(index)) {
           this.handleSelection(event, index);
         }
 
```

Comment:
The onChange event for the wrappedKeydown should be located inside the same if check as the handle selection similar to the wrappedOnClick above. Otherwise we will always call the onChange.